Ncsdk 19394 event manager section placement (#10093)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

  * New library.
    This was previously a component of the :ref:`nrf53_audio_app` application, now moved to :file:`lib/pcm_mix`.

* :ref:`app_event_manager`:

>   * Updated: The way the section names are created for event listeners allows the user to use any event naming scheme.
>     Before this update, to be sure event listerners sections does not overlap the event name should always have '_event' postfix.

Reviewer: ```suggestion
  * Updated the way section names are created for event listeners.
    This allows you to use any event naming scheme.
    Before this update, the event name must always have the ``_event`` postfix to make sure that event listener sections do not overlap.
```

  * New library.
    This was previously a component of the :ref:`nrf53_audio_app` application, now moved to :file:`lib/pcm_mix`.

* :ref:`app_event_manager`:

  * Updated the way section names are created for event subscribers.
    This allows you to use any event naming scheme.

>     For detailed information see the :ref:`NCSIDB-925 <ncsidb_925>` issue description on the :ref:`known_issues` page.

Reviewer: ```suggestion
    For more information, see the :ref:`NCSIDB-925 <ncsidb_925>` issue description on the :ref:`known_issues` page.
```

Reviewer 2: @rakons double-checked this as discussed and the ref seems to be working just fine :)

>     Before this update, the event name must always have the ``_event`` postfix to make sure that event listener sections do not overlap.

Author: For detailed information see the :ref:`NCSIDB-925 <ncsidb_925>` issue description on the :ref:`known_issues` page.
